Added disclaimer about 5930

ryanmrichard opened the pull request:
## Description
Updates the FNOCC documentation so that it includes a disclaimer about running in parallel on i7-5930 processors when compiling with Intel 15 and MKL 15.0.2

## Todos
Notable points that this PR has either accomplished or will accomplish.
- [x] Has not introduced further Fortran
- [x] Has warned the world of an impending evil lurking in their somewhat homely appearing CCSD correlation energy, caused by an apparent Intel bug

## Questions
- [ ]  Is the placement of the disclaimer satisfactory?

## Status
- [x]  Ready to go

loriab commented:
Maybe format it like this one https://github.com/psi4/psi4/blame/master/doc/sphinxman/source/sapt.rst#L18-L22 so the warning will show up in a nice red box and position after the page's intro. And make the psifour substitution pattern end in a vertical bar rather than square bracket. Thanks much, I think you've covered it.

ryanmrichard commented:
@loriab your suggested changes have been made.

loriab commented:
Sorry to nitpick, but your warning probably won't work unless the indentation lines up like in the example. Sphinx is picky. If you'd like to practice your interactive rebasing, you can squash your commits (http://docs.scipy.org/doc/numpy-1.10.1/dev/gitwash/development_workflow.html#rewriting-commit-history). You just have to force push the branch afterwards.

ryanmrichard commented:
I tried to edit the file on GitHub, and it then made me a branch, so I have been making these changes on GitHub and don't actually have a branch checked out, so I'll play with squashing later.

The warning was all on one line (I didn't do my usual 80 character shenanigans) so I suspect it may have worked.  Either way, I split the line and adjusted the indenting so that the lines start under "warning"

loriab commented:
Yes, it may have worked as one line. Thanks for fixing anyways. No problem on the squashing. Of course now I remember that MKL versioning is different from compiler versioning. So 15.0.3 and 16.0.2 are the compilers and something like 10.3 and 11.2 are the MKL. But I really don't remember the MKL versions. The new one is on a piece of paper on my desk. Sorry, didn't mean for this to be a saga.

loriab commented:
Ok, new compilers 16.0.2, new mkl 11.3.2. Old compilers 15.0.3, old mkl 11.2.3. https://software.intel.com/en-us/articles/which-version-of-the-intel-ipp-intel-mkl-and-intel-tbb-libraries-are-included-in-the-intel

ryanmrichard commented:
Ahh..that's my bad I had the MKL version right on the issue.  I just left it as Intel 15 and Intel 16 compilers because I think a few different versions have been thrown at it at this point.

loriab commented:
Due to our conversation, the #commits : #loc ratio got a little excessive. Since it's inconvenient to rebase this PR, I'm going to include the disclaimer in another commit, if that's ok @ryanmrichard .

ryanmrichard commented:
That's fine

Sent from my Android (without spellcheck).  Please forgive the brevity.
On Mar 28, 2016 4:06 PM, "Lori A. Burns" <notifications@github.com> wrote:

> Due to our conversation, the #commits : #loc ratio got a little excessive.
> Since it's inconvenient to rebase this PR, I'm going to include the
> disclaimer in another commit, if that's ok @ryanmrichard
> <https://github.com/ryanmrichard> .
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/psi4/psi4/pull/324#issuecomment-202561605>
>

dgasmith commented:
This branch became a bit complicated, Lori fixed this in #330 per discussion. Closing.
